editor/editor_node.cpp

```diff
@@ -1543,6 +1543,8 @@ void EditorNode::_save_scene(String p_file, int idx) {
 		return;
 	}
 
+	scene->propagate_notification(NOTIFICATION_EDITOR_PRE_SAVE);
```

Comment:
I'm not against the change but this needs design review (poke @godotengine/_editor @reduz).